src/appleseed/renderer/modeling/environmentedf/oslenvironmentedf.cpp

```diff
@@ -223,6 +513,76 @@ DictionaryArray OSLEnvironmentEDFFactory::get_input_metadata() const
                     .insert("shader_group", "Shader Groups"))
             .insert("use", "optional"));
 
+    metadata.push_back(
+        Dictionary()
+            .insert("name", "horizontal_shift")
+            .insert("label", "Horizontal Shift")
+            .insert("type", "numeric")
+            .insert("min",
+                Dictionary()
+                    .insert("value", "-360.0")
+                    .insert("type", "soft"))
+            .insert("max",
+                Dictionary()
+                    .insert("value", "360.0")
+                    .insert("type", "soft"))
+            .insert("default", "0.0")
+            .insert("use", "optional")
+            .insert("help", "Environment texture horizontal shift in degrees"));
+
+    metadata.push_back(
+        Dictionary()
+            .insert("name", "vertical_shift")
+            .insert("label", "Vertical Shift")
+            .insert("type", "numeric")
+            .insert("min",
+                Dictionary()
+                    .insert("value", "-360.0")
+                    .insert("type", "soft"))
+            .insert("max",
+                Dictionary()
+                    .insert("value", "360.0")
+                    .insert("type", "soft"))
+            .insert("default", "0.0")
+            .insert("use", "optional")
+            .insert("help", "Environment texture vertical shift in degrees"));
+
+    metadata.push_back(
+        Dictionary()
+            .insert("name", "importance_map_width")
```

Comment:
Maybe important_map_size?. We can then set width = size and height = size / 2.

Reply:
Agree. Done it like that.